wordpress-mobile/WordPress-iOS
# Renaming Site Details to Reader Blog to match Android

Fixes #6333 

To test:
Click a like in notifications, click a user, and verify the title of the screen is now "Reader Blog" instead of "Site Details"

Needs review: @aerych
